src/config.rs

```diff
@@ -56,7 +56,8 @@ impl NodeConfig {
                         DsnBuilder::gemini_3g()
                             .provider_storage_path(provider_storage_dir_getter()),
                     )
-                    .sync_from_dsn(true);
+                    .sync_from_dsn(true)
+                    .enable_subspace_block_relay(true);
```

Comment:
This option should ideally be removed from SDK

Reply:
@nazar-pc Agreed. I will open follow-up PR on monday to remove these options from builder and always passing true during configuration construction.